README.md

```diff
@@ -87,15 +87,33 @@ instance of the adage-server.
    ```
    docker-compose exec core python manage.py <management command> <arguments>
    ```
+
+   Note that the files that you will pass as some of these `<arguments>`
+   will be located in the `/srv/data/` folder, as specified by the `docker cp`
+   command above.
+
+   For example, if you wanted to copy a file on your computer called
+   'SomeOrganismAnnotation.tsv', and then load it into the instance's
+   database using the `import_data` management command, you would enter:
+
+   ```
+   docker cp \
+       <local path to file>/SomeOrganismAnnotation.tsv \
+       adage-django:/srv/data/
+
+   docker-compose exec core python manage.py import_data \
+       /srv/data/SomeOrganismAnnotation.tsv
+   ```
+
    These are the management commands currently available to load data files:
    `add_ml_model`,
    `create_or_update_participation_type`,
    `delete_participation_type`,
    `import_activity`,
    `import_data`,
    `import_gene_network.py`,
```

Comment:
We should remove the `.py` here, for consistency.

Reply:
Ah, yes, good catch! I hadn't noticed this.